basis_universal.lua

```diff
@@ -19,8 +19,15 @@ files {
   "*.c",
   "*.cpp",
 
+  "encoder/**.h",
+  "encoder/**.cpp",
+  "encoder/**.c",
+
   "transcoder/**.h",
   "transcoder/**.cpp",
+
+  "zstd/**.h",
```

Comment:
Looks like these are optional and may bite us if a security vulnerability is detected. We may also need to get legal approval if we want to use these. Instead, you can turn off zstd support with BASISD_SUPPORT_KTX2_ZSTD=0 as referenced https://github.com/Esri/basis_universal/commit/a8a1f7382457c9b76db824553839cf8127e225f3

Reply:
These are needed in order to be able to rebase our SLL runtimecore branch.